test/utils.test.js

```diff
@@ -36,3 +36,42 @@ describe("Find Highest of Numbers findHighest(...numbs)", () => {
     expect(findHighest(3,6,2,4,1)).to.equal(6);
   })
 })
+
+
+describe("Find Lowest of Numbers findLowest(...numbers)", () => {
```

Comment:
Lets add test for when we expect errors/excpetions to be thrown

Reply:
ok